### DISCUSSION TITLE
After the HA update, "Waste Collection Scheduler" no longer runs, the current HA update causes problems?

### DISCUSSION THREAD

**Discussion by Walwol** [mampfes/hacs_waste_collection_schedule#2207] [Q&A]
After the HA update, "Waste Collection Scheduler" no longer works. Is the current HA update causing problems?
This week I did the HA updates, after which "Waste Collection Scheduler" no longer worked. Lots of errors and it wouldn't start. So I uninstalled everything and am now trying to reinstall everything. But there is no installation via HACS.
Is this a known problem after the last HA update?

![grafik](https://github.com/mampfes/hacs_waste_collection_schedule/assets/142318285/c1d7e610-61af-460a-b95c-41ac814f6ccc)

![grafik](https://github.com/mampfes/hacs_waste_collection_schedule/assets/142318285/42722a06-f93e-43a1-b9bc-65692209f466)

**Comment by 5ila5** [comment]
[Removed] as not relevant

(just misunderstood: something my old message is not correct)

**Comment by Walwol** [comment]
### Solution for me found! 
In view of the different HA forums, I have seen that the same problem arises with some people. Therefore, it is likely to be a general problem with HACS in the HA. I also noticed this with my other installed HACS. So a general problem. 

- The problem also occurred after the last supervisor update in HA. 
- I did the following to solve the problem:

1. The network settings in HA adjusted as follows: 
![grafik](https://github.com/mampfes/hacs_waste_collection_schedule/assets/142318285/41aec989-5323-4770-ad2e-bde7795932cc)

- [ ] "Auto Configure" disabled!

2. HA reboot

**Now ALL runs as usual and the HACS are displayed faster and correctly, as well as the installation works again.** 

See also Solution on this Link:
[https://community.simon42.com/t/ich-kann-in-hacs-keine-repository-mehr-herunterladen/4339](url)
